// C-ERR

func main() {
    {
        var a = 1
        print(a)
    }

    print(a)
}